src/theme.ts

```diff
@@ -125,7 +125,8 @@ export class ThemeSet {
     const fnForWatch: Set<string> = new Set(found.map(f => path.resolve(f)))
 
     for (const f of fn) {
-      if (!hasMagic(f)) {
+      // globby's hasMagic (backed by fast-glob) always recognizes "\\" (Windows path separator) as the escape character.
+      if (!hasMagic(f.split(path.sep).join('/'))) {
```

Comment:
globby v11 provides `hasMagic()` backed by `fast-glob` instead of `node-glob`. `fast-glob` will recognize `\` (Windows path separator) as a magic character for escape, so we have to normalize it for watching theme directory correctly.